openfisca_core/tools/test_runner.py

```diff
@@ -138,6 +140,9 @@ def run_tests(
     if options.get('verbose'):
         argv.append('--verbose')
 
+    if workers is not None:
+        argv.append(f"--workers={workers[0]}")
```

Comment:
Why not default to `auto` and let users override that to `1` if they want so? 🙂 I don't see a reason to not use it if it is safe, and I don't see a reason to offer (let alone use) it if it is not safe.

Reply:
Yes workers are safe, yet we need to see how/if they work properly on windows. Otherwise I agree with you.

What really brings improvement however are to run several threads per process, yet out tests are not thread-safe :/